2022/ideas-list.md

```diff
@@ -2,11 +2,9 @@
 
 This is the home page of projects ideas of IOOS for Google Summer of Code 2022.
 
-|**Project**|**Description**|**links**|
-|--------|------------|-----|
-| erddapy | Separate erddapy into more functional layers, roughly following the [SQLAlchemy core/ORM](https://docs.sqlalchemy.org/en/14/intro.html#overview) model. | https://github.com/ioos/erddapy/issues/228 |
-| pyobis | Update the existing [pyobis](https://github.com/iobis/pyobis) python package to use the new OBIS API and ensure the package is usable for product generation in the future. | https://github.com/ioos/gsoc/issues/15 |
-| ioos_qc | | https://github.com/ioos/ioos_qc |
-| IOOS compliance checker | | https://github.com/ioos/compliance-checker |
-| kerchunk | Efficient access to IOOS data in the cloud with netCDF, Zarr, and JSON using [kerchunk](https://fsspec.github.io/kerchunk/).  | https://github.com/ioos/gsoc/issues/14 |
-| echoshader | Enhance the capability to interactively visualize large volumes of cloud-hosted ocean sonar data to accelerate the data exploration and discovery process. Expanding on [echopype](https://github.com/OSOceanAcoustics/echopype) | https://github.com/ioos/gsoc/issues/16 |
+|**Project** |**Description**|**links**| **hours** |
+|------------|---------------|---------|-----------|
+| erddapy    | Separate erddapy into more functional layers, roughly following the [SQLAlchemy core/ORM](https://docs.sqlalchemy.org/en/14/intro.html#overview) model.                                                                          | https://github.com/ioos/erddapy/issues/228 | 175  or 350  |
```

Comment:
Could you link erddapy in the description? https://github.com/ioos/erddapy

Reply:
Done! I also removed the SQLAlchemy doc link b/c that is not necessary here.